plugin-build/build.gradle.kts

```diff
@@ -61,6 +62,11 @@ dependencies {
     compileOnly(Libs.ASM)
     compileOnly(Libs.ASM_COMMONS)
 
+    implementation(kotlin("gradle-plugin-api"))
+    implementation("org.jetbrains.kotlin:kotlin-stdlib:${KotlinCompilerVersion.VERSION}")
```

Comment:
Do we need this one? I think it should come from the kotlin-gradle-plugin already